src/sources/util/http/prelude.rs

```diff
@@ -37,6 +37,15 @@ use super::{
 
 #[async_trait]
 pub trait HttpSource: Clone + Send + Sync + 'static {
+    fn enrich_events(
```

Comment:
I realize there is no documentation for this trait existing, but I think it is good practice to add it as we go.
In this case it is a public trait used by multiple sources so would be nice to have a doc comment.

Reply:
Just documented this in both traits that feature this function.